meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx6ullevk/lmp.cfg

```diff
@@ -51,6 +51,9 @@ CONFIG_CMD_XIMG=y
 CONFIG_SUPPORT_EMMC_RPMB=y
 CONFIG_SUPPORT_EMMC_BOOT=y
 CONFIG_DM_REGULATOR_FIXED=y
+CONFIG_CMD_SECONDARY_BOOT=y
+CONFIG_SECONDARY_BOOT_RUNTIME_DETECTION=y
+CONFIG_SECONDARY_BOOT_SECTOR_OFFSET=0x1000
 CONFIG_MMC_BROKEN_CD=y
```

Comment:
Looks like we'll need the CONFIG_SECONDARY_* items added to `lmp-base.cfg` as well to ensure we get a sit.bin when users build with `DISTRO=lmp-base` as they will be using the same `mfgtool` scripts to flash the HW.

Reply:
There is no lmp-base.cfg for imx6ullevk, should I use the common one (./meta-lmp-base/recipes-bsp/u-boot/u-boot-fio/lmp-base.cfg)?

Reply:
No need, we can have that in lmp-base.cfg once we add support for it (currently missing for imx6ullevk).

Reply:
But adding to lmp-base.cfg (apalis-imx6) will be required, so we can then use the same mfgtools files for both cases.